doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-ESP8266/index.md

```diff
@@ -117,7 +117,7 @@ You are free to set it to whatever you like.
 ## Writing applications
 
 After successful installation of requirements, you can proceed to application development.
-This section shows how to setup minimal Kaa application for ESP8266.
+This section shows how to setup minimal Kaa application for ESP8266. You can find an example code [here](attach/esp8266-sample/).
```

Comment:
This leads to the directory listing which is handled by the server. I'm not sure how github pages will handle that. (It may well forbid any directory listings.)